maintainers/maintainer-list.nix

```diff
@@ -11575,6 +11575,16 @@
     githubId = 1940568;
     name = "Sebastian Ball";
   };
+  seberm = {
+    email = "seberm@seberm.com";
+    github = "seberm";
+    githubId = 212597;
+    name = "Otto Sabart";
+    keys = [{
+      longkeyid = "rsa4096/0x1FA2DBE674387CC3";
```

Comment:
```suggestion
```
Otherwise ofborg raises this error: `error: The option lib.maintainers.seberm.keys.[definition 1-entry 1].longkeyid does not exist.`

Reply:
Removed, thank you!